packages/fast-web-utilities/src/numbers.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * This method keeps a given value within the bounds of a min and max value. If the value
+ * is larger than the max, the minimum value will be returned. If the value is smaller than the minimum,
+ * the maximum will be returned. Otherwise, the value is returned un-changed.
+ */
+export function wrapInBounds(min: number, max: number, value: number): number {
+    if (value < min) {
+        return max;
+    } else if (value > max) {
+        return min;
+    } else {
+        return value;
+    }
```

Comment:
don't actually need the last else, can just return the value.

Reply:
indeed.